src/_product-updates/2024-06-10-changelog-entry-page.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Changelog entry page
+tags: [New]
+image: /images/changelog/changelog-entry-page.png
+---
+
+![changelog-entry-page.png](/images/changelog/changelog-entry-page.png)
+
+We have recently improved the changelog to make it easier to share updates to your API documentation.
+
+In addition to the continuous deployment flow of the changelog, you can now click on each entry to generate an individual page for a specific deployment. This option allows you to easily share a link directly pointing to the changes you don't want your API consumers, whether they are members of your organization or partners, to miss.
```

Comment:
```suggestion
In addition to the continuous deployment flow of the changelog, you can now click on each entry to view an individual page for a specific change. This option allows you to easily share a link directly pointing to the changes you don't want your API consumers to miss, whether they are members of your organization or partners.
```

Reply:
we don't generate a specific page on click, the click only **shows** the new page.

Also I found the construction of the last sentence a bit hard to read to I think it's easier with the suggestion made above.

WDYT?

Reply:
I fully agree, it's indeed already "generated". Thanks for the other change, that makes the text way more understandable.

Thanks Paulr!